src/errors.rs

```
@@ -78,7 +80,7 @@ impl StdError for Error {
             ErrorKind::UTF8(ref err) => Some(err),
             ErrorKind::Token(_) => None,
             ErrorKind::InvalidCapability => None,
-            ErrorKind::WasmElement(ref err) => Some(err),
+            ErrorKind::WasmElement(_) => None,
```

Is there a reason why we decided to ignore this error?

---

There used to be a cause with parity wasm but there is no longer a cause other than the error description itself.